infra/scripts/azure-runner.sh

```diff
@@ -0,0 +1,158 @@
+
+    if ! time helm install --wait "$RELEASE" infra/charts/feast \
+        --timeout 15m \
+        --set "feast-jupyter.image.repository=${DOCKER_REPOSITORY}/feast-jupyter" \
+        --set "feast-jupyter.image.tag=${GIT_TAG}" \
+        --set "feast-online-serving.image.repository=${DOCKER_REPOSITORY}/feast-serving" \
+        --set "feast-online-serving.image.tag=${GIT_TAG}" \
+        --set "feast-jobservice.image.repository=${DOCKER_REPOSITORY}/feast-jobservice" \
+        --set "feast-jobservice.image.tag=${GIT_TAG}" \
+        --set "feast-core.image.repository=${DOCKER_REPOSITORY}/feast-core" \
+        --set "feast-core.image.tag=${GIT_TAG}" \
+        --set "prometheus-statsd-exporter.enabled=false" \
+        --set "prometheus.enabled=false" \
+        --set "grafana.enabled=false" \
+        --set "feast-jobservice.enabled=false" \
+        "$@" ; then
+
+        echo "Error during helm install. "
+        kubectl -n "$NAMESPACE" get pods
+
+        readarray -t CRASHED_PODS < <(kubectl -n "$NAMESPACE" get pods --no-headers=true | grep "$RELEASE" | awk '{if ($2 == "0/1") { print $1 } }')
+        echo "Crashed pods: ${CRASHED_PODS[*]}"
+
+        for POD in "${CRASHED_PODS[@]}"; do
+            echo "Logs from pod error $POD:"
+            kubectl -n "$NAMESPACE" logs "$POD" --previous
+        done
+
+        exit 1
+    fi
+}
+
+function setup_sparkop_role {
+    # Set up permissions for the default user in sparkop namespace so that Feast SDK can manage
+    # sparkapplication resources from the test runner pod.
+
+    cat <<EOF | kubectl apply -f -
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: use-spark-operator
+rules:
+- apiGroups: ["sparkoperator.k8s.io"]
+  resources: ["sparkapplications"]
+  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: use-spark-operator
+roleRef:
+  kind: Role
+  name: use-spark-operator
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: default
+EOF
+}
+
+GIT_TAG=$(git rev-parse HEAD)
+GIT_REMOTE_URL=$(git config --get remote.origin.url)
+
+echo "########## Starting e2e tests for ${GIT_REMOTE_URL} ${GIT_TAG} ###########"
+
+# This seems to make builds a bit faster.
+export DOCKER_BUILDKIT=1
+
+# Workaround for COPY command in core docker image that pulls local maven repo into the image
+# itself.
+mkdir .m2 2>/dev/null || true
+
+# Log into k8s.
+echo "${STEP_BREADCRUMB} Updating kubeconfig"
+az login --service-principal -u "$AZ_SERVICE_PRINCIPAL_ID" -p "$AZ_SERVICE_PRINCIPAL_PASS" --tenant "$AZ_SERVICE_PRINCIPAL_TENANT_ID"
+az aks get-credentials --resource-group "$RESOURCE_GROUP" --name "$AKS_CLUSTER_NAME"
+
+# Sanity check that kubectl is working.
+echo "${STEP_BREADCRUMB} k8s sanity check"
+kubectl get pods
+
+# e2e test - runs in sparkop namespace for consistency with AWS sparkop test.
```

Comment:
Is there a reason to keep these consistent? No conflict will occur?

Reply:
it's just so I can reuse the existing sparkop test code as is, and it needs to run in some namespace so why not "sparkop"